infrastructure/scheduler/schedule_dns_ingest.tf

```diff
@@ -4,10 +4,27 @@ resource "aws_cloudwatch_event_target" "schedule_dns_ingest" {
   input = "{}" # no info needed, just do the scan!
 }
 
+data "external" "current_minute" {
+  program = [
+    "python",
+    "-c",
+    # external datasource expects json, simple script to include now's minutes as the value in json
+    "import datetime; print(f\"{{\\\"min\\\":\\\"{datetime.datetime.now().minute}\\\"}}\")"
+  ]
+}
+
+locals {
+  # Originally this was always midnight, but we had multiple environments all trying to ingest
+  # at once which stops any from succeeding, this hack reduces the chances of that happening.
+  # tell it to kick off 5 mins after now to give things time to deploy
+  # TODO https://dsdmoj.atlassian.net/browse/SA-123 Share the ingest logic to avoid the duplication
+  ingest_schedule = "${tonumber(data.external.current_minute.result.min)} 0 * * ? *"
```

Comment:
this isn't 5 mins from now but 00:<current_min>

Reply:
I think I addressed this